Cargo.lock

```diff
@@ -1035,8 +1037,8 @@ dependencies = [
  "rustls-native-certs",
  "rustls-pemfile",
  "serde",
- "webpki",
- "webpki-roots",
+ "webpki 0.22.0",
+ "webpki-roots 0.22.1",
```

Comment:
This is fishy... we shouldn't introduce multiple versions of these crates

Reply:
I ran `cargo update` and that fixed this issue.